featuretools/primitives/__init__.py

```diff
@@ -1,2 +1,12 @@
 # flake8: noqa
 from .api import *
+
+import pkg_resources
+for entry_point in pkg_resources.iter_entry_points('featuretools_primitives'):
```

Comment:
add a comment to this block? something like 

"Load in primitives registered by other libraries into Featuretools namespace"

Reply:
Added